packages/zora/src/Zora.ts

```diff
@@ -282,11 +286,10 @@ export const simulateMint = async (
         functionName: 'mint',
         args: mintArgs,
         account: from,
-        // TODO: There's a bug in Viem preventing this behavior; log issue with them
-        // stateOverride: [{
```

Comment:
Not totally sure if there was a bug (at least in current version -- 2.9.23) since I was able to get this running fine